.Rprofile

```diff
@@ -1,4 +1,2 @@
 source("renv/activate.R")
-library(targets)
-targets::tar_load_globals()
```

Comment:
Was this causing problems, or just not necessary?

Reply:
I put the rationale under commit [6548eb0](https://github.com/Motivation-and-Behaviour/sleepIPD_analysis/pull/98/commits/6548eb0866de8609880f23ec3f8329976b5b2509)

TL;DR - it's being loaded by every worker, the docs [explicitly say](https://docs.ropensci.org/targets/reference/tar_load_globals.html#storage-access) not to do this.

Reply:
Ahh problematic, thanks (DR). Ideally there would be a way to get it to load when not executing the pipeline to make dev work seamless, but can see not good to mess with pipeline environments.